go/vt/vtctld/api.go

```diff
@@ -537,26 +488,7 @@ func initAPI(ctx context.Context, ts *topo.Server, actions *ActionRepository, he
 			return nil, fmt.Errorf("invalid tablet_health path: %q  expected path: /tablet_health/<cell>/<uid>", tabletPath)
 		}
 
-		if healthcheck == nil {
-			return nil, fmt.Errorf("healthcheck not initialized")
-		}
-
-		cell := parts[0]
-		uidStr := parts[1]
-		uid, err := topoproto.ParseUID(uidStr)
-		if err != nil {
-			return nil, fmt.Errorf("incorrect uid: %v", err)
-		}
-
-		tabletAlias := topodatapb.TabletAlias{
-			Cell: cell,
-			Uid:  uid,
-		}
-		tabletStat, err := healthcheck.GetTabletHealthByAlias(&tabletAlias)
-		if err != nil {
-			return nil, fmt.Errorf("could not get tabletStats: %v", err)
-		}
-		return tabletStat, nil
+		return nil, fmt.Errorf("healthcheck not initialized")
```

Comment:
Same here. If the endpoint is always returning an error, why would we even need the endpoint still? Can't we just remove it and have it return a 404 instead?

Reply:
sure, we can do that instead. @deepthi wdyt?

Reply:
404 sounds good to me

Reply:
this meant i had to change the function so i'd have access to the `ResponseWriter` to properly do the 404, but other than that, should be straightforward